components/sensor/sgp40.rst

```diff
@@ -0,0 +1,57 @@
+SGP40 Volatile Organic Compound Sensor
+==============================================
```

Comment:
too many = this must be the same length of title above

Reply:
Thats right, I thought travis checks should have caught this.